sheeprl/algos/a2c/utils.py

```diff
@@ -0,0 +1,49 @@
+from typing import Any, Dict
+
+import torch
+from lightning import Fabric
+
+from sheeprl.algos.a2c.agent import A2CAgent
+from sheeprl.utils.env import make_env
+
+AGGREGATOR_KEYS = {"Rewards/rew_avg", "Game/ep_len_avg", "Loss/value_loss", "Loss/policy_loss"}
```

Comment:
Can you please add the `MODELS_TO_REGISTER` variable? And also the configs in the `./sheeprl/configs/model_manager` folder.
Thanks